site/docs/guides/llm-redteaming.md

```diff
@@ -217,11 +225,19 @@ For example, `json.nested.output` will reference the output in the following API
 
 You can also reference nested objects. For example, `json.choices[0].message.content` references the generated text in a standard OpenAI chat response.
 
+:::tip
+Although the `targets` property supports multiple values, we recommend targeting a single HTTP endpoint per config.
```

Comment:
You don't need this tip here, you can offer clearer guidance by limiting the targets above